src/webvr-manager.js

```diff
@@ -42,6 +42,10 @@ var Wakelock = require('./wakelock.js');
 function WebVRManager(renderer, effect, params) {
   this.params = params || {};
 
+  // Give a unique to ID to each manager.
+  this.prefix = 'webvr';
```

Comment:
The logic is that the Manager should pass the uid to other things it might call (e.g. a player) which might use it to auto-construct CSS ids and classes. 
manager - webvr-298
player Id - webvr-298-player
player class - webvr-player
player caption Id (also aria-describedby attribute in player) - webvr-298-player-caption
player caption class - webvr-player-caption